test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -34,6 +34,7 @@
 #include "utilities/macros.hpp"
 #include "utilities/ostream.hpp"
 #include "unittest.hpp"
+#include "gmock/gmock.h"
```

Comment:
Is this a leftover?

Reply:
Oops, thank you. That slipped by me, it's from an earlier experiment.